### PR TITLE
Improved Deployment to make variables easier to change

### PR DESCRIPTION
**Pull request by Kroko-fant**
### Motivation and Context
Deployment changes in Orion are pretty complex as the plugin depends on external dependencies that often change versions.
To simplify these changes Orion now has all variables in a separate file so its easier to edit changes.

### Description
Gradle uses a properties file where variables can be defined. with the new deployment changes the variables got moved out of the build file to properties.

### Steps for Testing
Nothing in the plugin logic has changed. Changes were made in the plugins deployment therefor no manual testing is needed.

1. Install the release as described in [the readme](https://github.com/ls1intum/Orion/blob/main/README.md#testing-of-pull-requests)
2. Open an exercise in Orion
3. ...

#### Code Review
- [ ] Review 1
- [ ] Review 2

### CODE REVIEW

File: src/main/resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -1,8 +1,7 @@
 <idea-plugin url="https://artemis.ase.in.tum.de/">
     <id>de.tum.www1.orion</id>
     <name>Orion - Artemis Programming Exercise Integration</name>
-    <vendor email="alexander.ungar@tum.de" url="https://artemis.ase.in.tum.de">Technical University of Munich</vendor>
-    <description><![CDATA[
+    <vendor email="alexander.ungar@tum.de" url="https://artemis.ase.in.tum.de">Technical University of Munich</vendor>    <description><![CDATA[
```
Comment (lennart-keller) [inline src/main/resources/META-INF/plugin.xml:4]: Is this change intended? Loos off to me.

Comment (lennart-keller) [inline src/main/resources/META-INF/plugin.xml:4]: Is this change intended? Looks off to me with description being in the same line

**Review by lennart-keller — APPROVED** [review]
Code lgtm. Just one small question

File: src/main/resources/META-INF/plugin.xml (hunk 1 above)
Comment (Kroko-fant) [reply src/main/resources/META-INF/plugin.xml]: no it wasn't :D
